gms/gossiper.cc

```diff
@@ -389,6 +389,7 @@ future<> gossiper::do_send_ack2_msg(msg_addr from, utils::chunked_vector<gossip_
     gms::gossip_digest_ack2 ack2_msg(std::move(delta_ep_state_map));
     logger.debug("Calling do_send_ack2_msg to node {}, ack_msg_digest={}, ack2_msg={}", from, ack_msg_digest, ack2_msg);
     co_await ser::gossip_rpc_verbs::send_gossip_digest_ack2(&_messaging, from, std::move(ack2_msg));
+    logger.debug("do_send_ack2_msg finished");
```

Comment:
This function may be called concurrently for multiple endpoints, and then it will be hard to correlate this message to the corresponding `Calling do_send_ack2_msg` message.

Reply:
```
    log = await manager.server_open_log(failed_server.server_id)
    await log.wait_for("do_send_ack2_msg finished")
```

This log message is checked only for failed server.

Did I correctly understand your question?

Reply:
I'm not talking about the test. This logging will be executed everywhere, in particular on production clusters if we enable `--logger-log-level gossip=debug` (or `trace`).  And then if I want to see how `ack2` messages are exchanged, when they are finishing vs when they are starting, I will see a bunch of random `do_send_ack2_msg finished` messages appearing in the log, but I won't know which `Calling do_send_ack2_msg to node {}, ack_msg_digest={}, ack2_msg={}` messages they correspond to.

Reply:
i.e. bad user experience when browsing logs and trying to correlate events with each other.

For example suppose I will see
```
Calling do_send_ack2_msg to node 127.0.0.1
Calling do_send_ack2_msg to node 127.0.0.2
do_send_ack2_msg finished
do_send_ack2_msg finished
```

does the first `finished` message correspond to the 127.0.0.1 node or the 127.0.0.2 node? Cannot say. It will be even worse when there are more than two nodes

Reply:
Got that!
I updated the log to be: 
```
logger.debug("finished do_send_ack2_msg to node {}, ack_msg_digest={}, ack2_msg={}", from, ack_msg_digest, ack2_msg);
```

Lemme know if its fine or we should go via injection route

Reply:
much better, thanks